src/server/src/entities/documents.ts

```diff
@@ -0,0 +1,29 @@
+import {
+    Entity,
+    Column,
+    PrimaryGeneratedColumn,
+    ManyToOne,
+    BaseEntity,
+} from 'typeorm';
+import { Student } from '../entities/student';
+
+@Entity()
+export class Document extends BaseEntity {
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @Column({ nullable: false })
+    name: string;
+
+    @Column({ nullable: true })
+    isDefault: boolean;
+
+    @Column({ nullable: false})
+    dateAdded: Date;
+
+    @Column({ type: 'bytea', nullable: false })
+    document: Buffer;
+
+    @ManyToOne(() => Student, (student) => (student.documents))
```

Comment:
The student's class has no documents attribute. You need to specify the relationship in the student entity as well.

Reply:
I thought you said the online version has the documents attribute or we will have the documents attribute in the Student class?

Reply:
Sorry forgot to add that part. Added now